dev/devicelab/bin/tasks/android_plugin_example_app_build_test.dart

```diff
@@ -32,22 +32,47 @@ Future<void> main() async {
       options: <String>['--android', '--no-ios'],
     );
 
-    final Directory tempDir = Directory.systemTemp.createTempSync('flutter_plugin_test.');
-    final Directory projectDir = Directory(path.join(tempDir.path, 'plugin_test'));
+    final Directory tempDir =
+        Directory.systemTemp.createTempSync('flutter_plugin_test.');
+    final Directory projectDir =
+        Directory(path.join(tempDir.path, 'plugin_test'));
     try {
       await inDirectory(tempDir, () async {
         await flutter(
           'create',
-          options: <String>['--template=plugin', '--platforms=android', 'plugin_test'],
+          options: <String>[
+            '--template=plugin',
+            '--platforms=android',
+            'plugin_test'
```

Comment:
If you want to break it this way add a trailing comma

Reply:
+1